src/libraries/System.Security.Cryptography.Primitives/tests/CryptoStream.cs

```diff
@@ -317,6 +318,74 @@ public static void PaddedAes_PartialRead_Success()
             }
         }
 
+        [Fact]
```

Comment:
We should make this conditional on using a 64-bit process, given we're trying to allocate a 1.6GB array.

Reply:
Won't a 32-bit process just hit the try/OOM/skip?  My feeling was that the architecture doesn't really guarantee success or failure, so just accept "couldn't do it" and move on.

Or do you perhaps think 32-bit might get unfortunately close and then OOM after that point?

Reply:
I'm more worried it succeeds and then causes other tests in the process to start OOM'ing.